<?xml version="1.0"?>
<launch>

  <arg name="base_serial_port" default="/dev/ttyACM0"/>
  <arg name="rover_serial_port" default="/dev/ttyACM1"/>
  <arg name="base_host" default="localhost"/>
  <arg name="rover_host" default="localhost"/>

	<group ns="base">

		<node name="f9p" pkg="ublox" type="ublox_ros">
			<param name="serial_port" value="/dev/ttyACM0"/>
			<param name="base_type" value = "moving"/>  <!--moving or stationary-->
			<param name="rover_quantity" value = "0"/>     <!--number of rovers-->
			<param name="log_filename" value="/home/tpool2/sfrbxlog.txt"/>
		</node>
	</group>

</launch>
